Python docs (#1570)
superfly/docs: python/the-basics/initial-setup.html.md

---

> title: "Setting up a Python Environment"

Reviewer: Just a heads up, it was a bit confusing for me to have this page (and the `Multi-stage Builds` page) as a link. It wasn't intuitive that those were other clickable pages.

Reviewer: It became more clear when I accessed other main pages.

Author: that's a fair point; maybe we can bring this up as a design thing? The same issue will apply to js and rust docs too as they follow the same pattern

Reviewer: Yes, they can suggest some changes/improvements in general 😁

layout: framework_docs
objective: How to setup a functional python environment on your local machine
sitemap: false
order: 0
---

## Initial Local Setup

Make sure that [Python](https://www.python.org/) is already installed on your computer along with a way to create virtual environments.

This allows you to run your project locally, and test that it works, before deploying it to Fly.io.

<section class="callout">
We recommend the latest [supported versions](https://devguide.python.org/versions/#supported-versions) of Python.
</section>

## Dependency Management

> For project and dependency management we use [Poetry](https://python-poetry.org/). It comes with a nice developer experience and combines multiple tools in one resembling a lot of other package managers out there. 

Reviewer 2: ```suggestion
For project and dependency management we use [Poetry](https://python-poetry.org/). Like most package managers, Poetry combines multiple tools in one. 
```
